spec/vmpooler/api/v1_spec.rb

```diff
@@ -332,6 +341,23 @@ def app()
           expect(last_response.status).to eq(400)
         end
 
+      context '(allowed_tags configured)' do
+        let(:config) { {
+          config: {
+            'allowed_tags' => ['created_by', 'project', 'url']
+          }
+        } }
+
+        it 'fails if restricted tag is specified' do
```

Comment:
nit: The wording here is a little confusing, when I read it I thought that it implied there is some way to explicitly name restricted tags when it's actually just preventing the use of tags other than what are explicitly allowed.